builders/build.yaml

```diff
@@ -70,7 +71,8 @@ steps:
   - 'go'
   - 'build'
   - '-trimpath'
-  - '-ldflags=-s -w -X=${_REPO}/internal/buildinfo.BuildID=${BUILD_ID} -X=${_REPO}/internal/buildinfo.BuildTag=${_TAG} -extldflags=-static'
+  - &add-buildinfo
```

Comment:
AFAIK, Cloud Build does not support the full YAML spec: b/152180846. That's largely why there's so much copy-paste in these files.

Reply:
TIL :-(